src/backend/Types.h

```diff
@@ -64,6 +64,7 @@ struct NFT
     ripple::uint256 tokenID;
     std::uint32_t ledgerSequence;
     ripple::AccountID owner;
+    std::optional<Blob> uri;
```

Comment:
why optional?

Reply:
NFTs don't have to have a URI at all

Reply:
make it not optional and throw an error in the fetcher